products/amazon-cdk.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Amazon CDK
+category: framework
+iconSlug: amazonaws
+permalink: /amazon-cdk
+changelogTemplate: https://github.com/aws/aws-cdk/releases/tag/v__LATEST__
+releaseDateColumn: true
+
+auto:
+-   git: https://github.com/aws/aws-cdk
+
+identifiers:
+-   repology: aws-cdk
+-   purl: pkg:npm/aws-cdk
+
+releases:
+-   releaseCycle: "2.103"
+    releaseDate: 2023-10-26
+    support: true
+    eol: false
+    latest: "2.103.1"
+    latestReleaseDate: 2023-10-27
+-   releaseCycle: "2.102"
+    releaseDate: 2023-10-19
+    support: false
+    eol: true
+    latest: "2.102.1"
+    latestReleaseDate: 2023-10-25
```

Comment:
Since cdk follows a very short development cycle (weekly) and the older version is unsupported immediately on the next release, it is better to squish it to a single row without any loss of information.

This also ensures that automatic updates are seamless here, till a v3 is released.

```suggestion
-   releaseCycle: "2"
    releaseDate: 2021-12-02
    support: true
    eol: false
    latest: "2.103.1"
    latestReleaseDate: 2023-10-27
```

I considered adding v1 as a separate row, but v2 was the first stable release.